src/api/uploads/FolderUploadNode.js

```diff
@@ -66,8 +70,8 @@ class FolderUploadNode {
      */
     uploadChildFolders = async (errorCallback: Function) => {
         // $FlowFixMe
-        const folderNodes: Array<FolderUploadNode> = Object.values(this.folders);
-        const promises = folderNodes.map((folder) => folder.upload(this.folderId, errorCallback));
+        const folders: Array<FolderUploadNode> = Object.values(this.folders);
```

Comment:
Feels like we have a lot of flow fix stuff

Reply:
lol yeah I'll try to cast them instead